src/modules/esl-utils/dom/traversing.ts

```diff
@@ -13,6 +13,11 @@ export const createSequenceFinder = (next: (el: Element) => Element | null) => {
   };
 };
 
+/** Checks if element matches passed selector or exact predicate function */
+export const isMatches = (el: Element, matcher?: string | ((el: Element) => boolean)): boolean => {
+  return (typeof matcher === 'string' && el.matches(matcher) || typeof matcher === 'function' && matcher.call(el));
+};
```

Comment:
Matcher predicate still not called according it's signature.
Consider Unit tests to check `isMatches` utility proper work.

Reply:
Still not :)
```
const sample = document.createElement('div');
expect(isMatches(sample, (el) => sample === el)).toBe(true);
expect(isMatches(sample, 'div').toBe(true);
expect(isMatches(sample, 'span').toBe(true);
```